### PR TITLE
Edited the formatting of the Overdue and Adherence lists

### PR DESCRIPTION
**Pull request by dburka**
**Story card:** NONE

## Because

The formatting of the new Overdue list filters was a bit funky

## This addresses

Basic formatting issues. Nothing structural.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -561,380 +563,3 @@ Subnav
   font-weight: bold;
   cursor: pointer;
 }
-/*****************
-Dashboard UI
-*****************/
```
Comment (harimohanraj89) [inline app/assets/stylesheets/application.scss:566]: This chunk moved to `_dashboard.scss`

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -561,380 +563,3 @@ Subnav
...
-
-.table-compact .row-total td {
-    background: #FFFAE3;
-    font-weight: bold;
-    border-color: #E2DBBB;
-}
-
-/******************
-Overdue list
-******************/
```
Comment (harimohanraj89) [inline app/assets/stylesheets/application.scss:839]: This chunk moved to `_overdue.scss`

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -561,380 +563,3 @@ Subnav
...
-    font-size: 12px;
-    color: $grey-mid;
-    border-top: 1px solid $grey-mid;
-    border-bottom: 1px solid $grey-mid;
-    font-weight: bold;
-}
-
-/******************
-Breadcrumb nav
-******************/
```
Comment (harimohanraj89) [inline app/assets/stylesheets/application.scss:892]: This chunk and "Environments" below moved to `_navigation.scss`

File: app/views/layouts/overdue.html.erb [hunk 4]
```diff
@@ -79,19 +79,20 @@
         <% end %>
       </section>
 
-      <section>
+      <section class="mt-5">
         <% if current_facility %>
           <% if @patient_summaries.present? && policy([:overdue_list, @patient_summaries.first]).download? %>
+            <h4>Download</h4>
             <%= link_to(@index_params.merge(format: "csv"), class: "", data: { confirm: I18n.t('admin.phi_download_alert') }) do %>
-              <h4>Download Results</h4>
```
Comment (rsanheim) [inline app/views/layouts/overdue.html.erb:86]: We lost the anchor text here with this change. 😁 I'll create a PR to get it back.

**Pull request merged**